tensorrec/tensorrec.py

```diff
@@ -421,8 +423,12 @@ def fit_partial(self, interactions, user_features, item_features, epochs=1, lear
                                                            user_features=user_features,
                                                            item_features=item_features,
                                                            user_batch_size=user_batch_size,
-                                                           extra_feed_kwargs={self.tf_learning_rate: learning_rate,
-                                                                              self.tf_alpha: alpha})
+                                                           extra_feed_kwargs={self.tf_learning_rate: learning_rate})
+
+        # This scales down the alpha based on the number of batches and inserts the new alpha in all feed dicts
+        batched_alpha = calculate_batched_alpha(num_batches=len(batched_feed_dicts), alpha=alpha)
+        for feed_dict in batched_feed_dicts:
```

Comment:
Only issue I see here, maybe an edge case (or not even possible to change) but (i think) this fixes the order of batched_feed_dicts for training. Any concern that someone somehow might mess with the batch order?

Reply:
I think messing with batch order would be okay. In fact, we may want to add `shuffle_batches` as a fit arg to intentionally shuffle around the batches every epoch.

Reply:
:+1: